### ISSUE TITLE
Forbid resources/policies creation with zone label for global

### ISSUE THREAD

**Issue by Icarus9913** [kumahq/kuma#10590]
### What happened?

Do we need to reinforce our validation for global to forbid those resources/policies creation with `kuma.io/origin: zone` ?

**Comment by jakubdyszkiewicz** [comment]
Yes. Baptiste hit this problem and it caused the confusion. Global was on Kube, we were applying resources with `kuma.io/origin: zone` which broke policy matching.

**Comment by lahabana** [comment]
We should also ensure that federation doesn't expose with zone labels so that this doesn't break the federation flow
